package.json

```diff
@@ -1,11 +1,8 @@
 {
   "name": "codeworks-netflix-react-exercise",
```

Comment:
![busted](https://i.giphy.com/media/2yWxpmYVjpVQY/giphy.webp)

busted

Reply:
nah just kidding, it's not that much of a problem, but try to start a full-stack system by yourself, it's a great practice.